src/components/Incidents/AlertsChart/AlertsChart.jsx

```diff
@@ -1,140 +1,167 @@
 import * as React from 'react';
 
-import { Chart, ChartAxis, ChartBar, ChartGroup, ChartLabel, ChartThemeColor } from '@patternfly/react-charts';
+import {
+  Chart,
+  ChartAxis,
+  ChartBar,
+  ChartGroup,
+  ChartLabel,
+  ChartThemeColor,
+} from '@patternfly/react-charts';
 import { VictoryZoomContainer } from 'victory-zoom-container';
 import { Card, CardTitle } from '@patternfly/react-core';
 
-const days = ['Jul 25', 'Jul 26', 'Jul 27', 'Jul 28', 'Jul 29', 'Jul 30', 'Jul 31', 'Aug 1', 'Aug 2', 'Aug 3', 'Aug 4', 'Aug 5', 'Aug 6', 'Aug 7', 'Today',]
+const days = [
+  'Jul 25',
+  'Jul 26',
+  'Jul 27',
+  'Jul 28',
+  'Jul 29',
+  'Jul 30',
+  'Jul 31',
+  'Aug 1',
+  'Aug 2',
+  'Aug 3',
+  'Aug 4',
+  'Aug 5',
+  'Aug 6',
+  'Aug 7',
+  'Today',
+];
 const AlertsChart = () => {
```

Comment:
It seems that this component is the same as Incidents chart. Would you say they can be reused or they will diverge enough in order to have 2 separate components?